Assets/Scripts/Gamestate/Round.cs

```diff
@@ -75,9 +75,9 @@ public int KillCount(PlayerManager player)
         return kills[player].Count;
     }
 
-    public bool IsWinner(PlayerManager player)
+    public bool IsWinner(PlayerIdentity player)
     {
-        return player == winner;
+        return player == Winner;
```

Comment:
I know why it is like this, but this _will_ look weird to anyone else 😛

Reply:
Agreed